sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/StorageException.java

```diff
@@ -23,7 +23,7 @@
     private final StorageErrorCode errorCode;
     private final String message;
 
-    StorageException(StorageErrorException e, String responseBody) {
+    public StorageException(StorageErrorException e, String responseBody) {
```

Comment:
all public methods would be asked to have Javadocs, I would expect checkstyle fails in here

Reply:
I believe that check missing constructors, I'll add Javadocs